airflow-core/src/airflow/ui/src/i18n/locales/zh-TW/dashboard.json

```
@@ -3,11 +3,21 @@
   "health": {
     "dagProcessor": "Dag 處理器",
     "health": "健康狀態",
+    "healthy": "健康",
+    "lastHeartbeat": "最後心跳",
     "metaDatabase": "系統資料庫",
     "scheduler": "排程器",
-    "triggerer": "觸發器"
+    "status": "狀態",
+    "triggerer": "觸發器",
+    "unhealthy": "不健康"
```

I get the point of this but ...

how about

```suggestion
    "unhealthy": "健康狀態異常"
```

?

---

Sure, makes sense.